test/core/exceptions.wast

```diff
@@ -0,0 +1,429 @@
+;; Test exception handling support
```

Comment:
Can you split this up into files focussing on individual instructions?

throw.wast
rethrow.wast
try_catch.wast
try_delegate.wast

Reply:
Done.